toolchains/iOS.framework.cmake

```diff
@@ -3,7 +3,7 @@ include(${CMAKE_SOURCE_DIR}/toolchains/iOS.toolchain.cmake)
 message(STATUS "Build for iOS archs " ${CMAKE_OSX_ARCHITECTURES})
 
 set(FRAMEWORK_NAME TangramMap)
-set(FRAMEWORK_VERSION "1.0")
```

Comment:
There is no general concept of `SNAPSHOT` releases in iOS land. If you wanted to do something like that, you could use `ALPHA` or `BETA` tags.